test/e2e/common_suite.go

```diff
@@ -16,34 +16,31 @@ import (
 	log "github.com/sirupsen/logrus"
 	v1 "k8s.io/api/core/v1"
 	"sigs.k8s.io/e2e-framework/pkg/env"
-	envconf "sigs.k8s.io/e2e-framework/pkg/envconf"
 )
 
+const E2eNamespace = "coco-pp-e2e-test"
```

Comment:
If we change the `const` to the `var`
```
var E2eNamespace = envconf.RandomName("coco-pp-e2e-test", 20)
```
can make the E2eNamespace be random to `coco-pp-e2e-test-xxx`

Reply:
I'm on the fence if having a unique namespace name per e2e run is actually required, but I don't see any harm in doing it, so I'll update this. Thanks